plugins/pipx/plugin.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "pipx",
+  "version": "0.1.0",
+  "description": "Installs Pipx and makes it available",
+  "packages": ["pipx@latest"],
```

Comment:
This will be our first plugin that includes a package, up until a few months ago devbox plugins only supported config.

_I think_ this is a good use case for bundling a package in a plugin, but just keep an eye out for surprising behaviour - i.e. I'm not sure how the version of `pipx` is set and updated.

Reply:
yes good call. I think it might make sense to at least pin to a pipx version so that it's repeatable (my guess would be otherwise this is resolved at the app level meaning you could run it twice with different results as it stands)